common/aws.go

```diff
@@ -0,0 +1,3 @@
+package common
+
+var AwsPartitions = []string{"aws", "aws-cn", "aws-us-gov"}
```

Comment:
it's not necessary to be in the `common` - just create a separate file with constants/variables inside `aws` directory`

Reply:
Moved to `aws/constants.go`